build/centos7-puppet6/Puppetfile

```diff
@@ -44,7 +44,8 @@
 #   └── saz-sudo (v6.0.0)
 forge "https://forgeapi.puppetlabs.com"
 
-mod 'ghoneycutt-facter'
+# mod 'ghoneycutt-facter'
```

Comment:
```suggestion
```
Would be cleaner to remove the line, considering the dependency is pinned now.
Perhaps makes sense to mention in a comment the new reason for pinning it.